lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -15,6 +15,8 @@ foam.CLASS({
     'org.chromium.webidl.URLExtractor',
   ],
 
+  imports: [ 'outputDao? as dao' ],
```

Comment:
nit: `dao` is often used as a property for components that have their main data provided by a dao. It would be easy to mistakenly write `myRunner.dao.` when one meant `myRunner.outputDAO.`. Could we rename this to `ctxOutputDAO`?